packages/amplify-cli-core/API.md

```diff
@@ -1565,6 +1585,17 @@ export type WriteCFNTemplateOptions = {
 
 // Warnings were encountered during analysis:
 //
+// src/context/context-extensions.ts:257:19 - (ae-forgotten-export) The symbol "info" needs to be exported by the entry point index.d.ts
+// src/context/context-extensions.ts:257:19 - (ae-forgotten-export) The symbol "fancy" needs to be exported by the entry point index.d.ts
+// src/context/context-extensions.ts:257:19 - (ae-forgotten-export) The symbol "warning" needs to be exported by the entry point index.d.ts
+// src/context/context-extensions.ts:257:19 - (ae-forgotten-export) The symbol "error" needs to be exported by the entry point index.d.ts
+// src/context/context-extensions.ts:257:19 - (ae-forgotten-export) The symbol "success" needs to be exported by the entry point index.d.ts
+// src/context/context-extensions.ts:257:19 - (ae-forgotten-export) The symbol "table" needs to be exported by the entry point index.d.ts
+// src/context/context-extensions.ts:257:19 - (ae-forgotten-export) The symbol "debug" needs to be exported by the entry point index.d.ts
+// src/context/context-extensions.ts:257:19 - (ae-forgotten-export) The symbol "green" needs to be exported by the entry point index.d.ts
+// src/context/context-extensions.ts:257:19 - (ae-forgotten-export) The symbol "yellow" needs to be exported by the entry point index.d.ts
+// src/context/context-extensions.ts:257:19 - (ae-forgotten-export) The symbol "red" needs to be exported by the entry point index.d.ts
+// src/context/context-extensions.ts:257:19 - (ae-forgotten-export) The symbol "blue" needs to be exported by the entry point index.d.ts
```

Comment:
are these warnings legit ?

Reply:
I am not sure what is causing those. I've worked through a few ways to try to get rid of them, but was not successful. The `print` is exported and its members are accessible in outside packages. I tried scoping the exports from `context-extensions` and that did not work either.

See https://github.com/sdstolworthy/amplify-cli/blob/e1111304ffc89d7bec691df85823c331a4e8e50a/packages/amplify-cli/src/plugin-manager.ts#L140

Reply:
I just updated the `$TSContext` with a more meaningful type than `any`

Reply:
Looks like either these (see screenshot) should get exported.
Or (better?) `export const print` should get properties typed as `(message: string) => void` (or so) and those private functions assigned to them.

![image](https://user-images.githubusercontent.com/5849952/217401552-f8ff3f50-a465-499d-b47c-f9ac55dea982.png)

Reply:
Thanks. Did that. Those warnings are removed.